lib/editor/atto/plugins/kalturamedia/ltibrowse_container.php

```diff
@@ -19,3 +38,6 @@ function kaltura_atto_embed(data) {
         buttonJs.embedItem(buttonJs, data);
     }
 </script>
+
+<?php
+echo $OUTPUT->footer();
```

Comment:
Is this necessary?

Reply:
This way you actually print the complete HTML page. It will work without it but I believe that things should be done properly.

Reply:
Couldn't agree more.
I just wanted to make sure that it doesn't add unnecessary stuff to the page which can potentially break it.

Thank you very much!